packages/story-editor/src/components/library/panes/media/media3p/karma/mediaFetching.karma.js

```diff
@@ -335,160 +311,133 @@ xdescribe('Media3pPane fetching', () => {
     expect(document.activeElement.textContent).toBe('Sustainability');
 
     await fixture.events.keyboard.press('ArrowDown');
-    const expandButton = fixture.screen.getByRole('button', { name: 'Expand' });
-    expect(expandButton.getAttribute('aria-expanded')).toBe('true');
+    expect(
+      fixture.editor.library.media3p.expandFiltersButton.getAttribute(
+        'aria-expanded'
+      )
+    ).toBe('true');
   });
 
   it('should fetch 2nd page', async () => {
-    mockListMedia();
     await fixture.events.click(fixture.editor.library.media3pTab);
 
-    const mediaGallery = unsplashSection.querySelector(
-      '[data-testid="media-gallery-container"]'
+    const mediaGallery = fixture.editor.library.media3p.mediaGallery;
+
+    await expectMediaElements(
+      fixture.editor.library.media3p.unsplashSection,
+      MEDIA_PER_PAGE
     );
-    await expectMediaElements(unsplashSection, MEDIA_PER_PAGE);
 
     mediaGallery.scrollTo(
       0,
       mediaGallery.scrollHeight - mediaGallery.clientHeight - ROOT_MARGIN / 2
     );
-    jasmine.clock().tick(500);
-    await expectMediaElements(unsplashSection, MEDIA_PER_PAGE * 2);
-  });
-
-  // The scroll position was being reset because the resize event in <Gallery>
-  // would render no images when the media pane was hidden (width->0).
-  // This was fixed by re-rendering <Gallery> whenever MediaPane/Media3pPane is
-  // re-rendered, which causes the resize event to be suppressed at exactly the
-  // right time.
-  // A more robust fix to the scroll position reset issue can be found here:
-  // https://github.com/neptunian/react-photo-gallery/pull/184
-  // If that PR be released, another option is to patch it using:
-  // https://www.npmjs.com/package/patch-package
-  it('should retain scroll position on tab change', async () => {
```

Comment:
This doesn't happen in the app anymore. Is this a regression?
<img src="https://user-images.githubusercontent.com/22185279/142277264-d331eb71-cf38-4c7a-80c6-042f0913f5c1.gif" height="500px" />

Reply:
@miina ^^

Reply:
Probably caused by the panel contents not rendering anymore when not active.

Also not aware of this ever being feature / requirement, so I think it's OK to remove this test.